### PR TITLE
feat(core-api): add additional fields to bridgechains search schema

### LINKED ISSUE #3164: [2.6][T4] extend bridgechains search endpoint
The `bridgechains/search` endpoint only supports a `bridgechainId` in its payload schema. This should be extended to also support searching by `businessId`, `name`, `seedNodes`, `genesisHash` and `bridgechainRepository`. 

@faustbrian, yes, i can submit a PR :trollface:

### PR DESCRIPTION
**Pull request by dated**
## Summary

Closes #3164.

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: packages/core-api/src/handlers/bridgechains/schema.ts [hunk 1]
```diff
@@ -35,5 +35,16 @@ export const search: object = {
...
+        businessId: Joi.number()
+            .integer()
+            .min(1),
+        genesisHash: Joi.string()
+            .hex()
+            .length(64),
+        name: Joi.string()
+            .regex(/^[a-zA-Z0-9_-]+$/)
+            .max(40),
+        seedNodes: Joi.string().ip(),
```
Comment (supaiku0) [inline packages/core-api/src/handlers/bridgechains/schema.ts]: Shouldn't this accept an array or be named singular?

Comment (dated) [reply packages/core-api/src/handlers/bridgechains/schema.ts]: Indeed, sorry about that. Will be fixed.

**Comment by dated** [comment]
I've added a new filter, `every`, which returns `true` if all values of the given array are included in the checked property @supaiku0.

File: packages/core-database/src/repositories/utils/filter-rows.ts [hunk 2]
```diff
@@ -68,6 +68,23 @@ export = <T = any>(rows: ReadonlyArray<T>, params: Database.IParameters, filters
             }
         }
 
+        if (filters.hasOwnProperty("every")) {
+            for (const elem of filters.every) {
+                if (params[elem] && getProperty(item, elem)) {
+                    if (Array.isArray(item[elem])) {
+                        if (Array.isArray(params[elem])) {
+                            // @ts-ignore
+                            return params[elem].every(a => item[elem].indexOf(a) > -1);
```
Comment (supaiku0) [inline packages/core-database/src/repositories/utils/filter-rows.ts]:
```suggestion
                            return params[elem].every(a => item[elem].includes(a));
```

**Pull request merged**